test/celeritas/field/LinearPropagator.test.cc

```diff
@@ -99,7 +99,7 @@ TEST_F(LinearPropagatorTest, all)
 
     // Check state
     EXPECT_VEC_SOFT_EQ(Real3({30, 0, 20}), geo.pos());
-    EXPECT_EQ("si_tracker", this->volume_label(geo));
+    EXPECT_EQ("vacuum_tube", this->volume_label(geo));
```

Comment:
Why did this change??

Reply:
I had added a `cross_boundary()` call after each `move_to_boundary()` call in the LinearPropagator::operator() code.
Then I tried to revert that on the other commit, but it looks like something did not go well.  I will fix this.

Reply:
Reverted.